test/amp_tags_test.js

```diff
@@ -13,18 +13,30 @@ describe('AmpTags', function () {
     "sketches-host": "https://madrid.quintype.io",
   };
 
-  it("gets amp tags for supported stories", function () {
+  it("gets amp tags for supported stories other than visual stories", function () {
     const story = { slug: "section/slug", "is-amp-supported": true };
     const string = getSeoMetadata(seoConfig, config, "story-page", { data: { story: story } }, {});
     assertContains('<link rel="amphtml" href="/amp/story/section%2Fslug"/>', string);
   });
 
+  it("gets amp tags only for supported visual stories", function () {
+    const story = { slug: "section/slug", "is-amp-supported": true };
+    const string = getSeoMetadata(seoConfig, config, "story-page", { data: { story: { ...story, "story-template": "visual-story" } } }, {});
+    assertContains('<link rel="amphtml" href="/section%2Fslug"/>', string);
+  });
+
   it("does not rely on is-amp-supported in story API", function () {
     const story = { slug: "section/slug", "is-amp-supported": false };
     const string = getSeoMetadata(seoConfig, config, "story-page", { data: { story: story } }, {});
     assertContains('<link rel="amphtml" href="/amp/story/section%2Fslug"/>', string);
   });
 
+  it("does not rely on is-amp-supported in visual story API", function () {
+    const story = { slug: "section/slug", "is-amp-supported": false };
+    const string = getSeoMetadata(seoConfig, config, "story-page", { data: { story: { ...story, "story-template": "visual-story" } } }, {});
+    assertContains('<link rel="amphtml" href="/section%2Fslug"/>', string);
+  });
+
```

Comment:
This test isn't needed. `is-amp-supported` is independent of story template

Reply:
Removed.